examples/oscilator_example_g_stewart.py

```diff
@@ -0,0 +1,100 @@
+import nosnoc
+from casadi import SX, vertcat, horzcat
+import numpy as np
+import matplotlib.pyplot as plt
+
+OMEGA = 2 * np.pi
+A1 = np.array([[1, OMEGA], [-OMEGA, 1]])
+A2 = np.array([[1, -OMEGA], [OMEGA, 1]])
+R_OSC = 1
+
+
+def get_oscilator_model():
+
+    # Initial Value
+    x0 = np.array([np.exp([-1])[0], 0])
+
+    # Variable defintion
+    x1 = SX.sym("x1")
+    x2 = SX.sym("x2")
+    x = vertcat(x1, x2)
+    # every constraint function corresponds to a sys (note that the c_i might be vector valued)
+    c = [x1**2 + x2**2 - R_OSC**2]
+    # sign matrix for the modes
+    S = [np.array([[1], [-1]])]
+
+    f_11 = A1 @ x
+    f_12 = A2 @ x
+    # in matrix form
+    F = [horzcat(f_11, f_12)]
+
+    g_Stewart_list = [-S[i] @ c[i] for i in range(1)]
+    model = nosnoc.NosnocModel(x=x, F=F, g_Stewart=g_Stewart_list, x0=x0)
```

Comment:
can you merge this as an option in the original file?
I think just these 2 lines are different, so it would be more compact.